Project1/application.py

```diff
@@ -62,8 +62,8 @@ def authenticate() :
         if user :
             salt = user.password[:64]
             stored_password = user.password[64:]
-            # pwdhash = hashlib.pbkdf2_hmac('sha512', password.encode('utf-8'), salt.encode('ascii'), 100000)
-            # pwdhash = binascii.hexlify(pwdhash).decode('ascii')
+            pwdhash = hashlib.pbkdf2_hmac('sha512', password.encode('utf-8'), salt.encode('ascii'), 100000)
+            pwdhash = binascii.hexlify(pwdhash).decode('ascii')
```

Comment:
use meaningfull variable names

Reply:
ok ma'am

Reply:
can I merge ma'am

Reply:
Ma'am pwdhash means changing password to hash. It is taken from divya's code base. If I change now merge conflicts will arise